packages/utils/src/signingUtils.ts

```diff
@@ -1,95 +1,271 @@
-/* eslint-disable @typescript-eslint/no-extraneous-class */
+/* eslint-disable class-methods-use-this */
 import secp256k1 from 'secp256k1'
 import { Keccak } from 'sha3'
 import { ml_dsa87 } from '@noble/post-quantum/ml-dsa'
-import { areEqualBinaries } from './binaryUtils'
-import { UserIDRaw } from './UserID'
 import { randomBytes } from '@noble/post-quantum/utils'
+import { p256 } from '@noble/curves/p256'
+import { areEqualBinaries, binaryToHex } from './binaryUtils'
+import { UserIDRaw } from './UserID'
+import { getSubtle } from './crossPlatformCrypto'
+import { webcrypto } from 'crypto'
+
+export const SUPPORTED_KEY_PAIR_TYPES = [
```

Comment:
`SUPPORTED_` prefix would indicate that we have some specific unsupported key types

```suggestion
export const KEY_PAIR_TYPES = [
```

Reply:
If we rename `KeyPairType` to `KeyType` as per your suggestion, and since this is an array consisting of the different `KeyType` values, I guess the name of the array should then be `KEY_TYPES`?